### PR TITLE
sample(card) - Adding styling sample for cards

### PR DESCRIPTION
**Pull request by astaev**
https://github.com/IgniteUI/igniteui-docfx/issues/1275

### CODE REVIEW

File: src/app/layouts/card/card-styling-sample/card-styling-sample.component.scss [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+/*ENHANCED APPROACHES*/
+
+// $yellow-color: #e7eb0b;
+// $black-color: #f00b1e;
+// $dark-palette: igx-palette($primary: $black-color, $secondary: $yellow-color);
+
+/*PALETTE and IGX-COLOR*/
+
+// $dark-card: igx-card-theme(
```
Comment (tachojelev) [inline src/app/layouts/card/card-styling-sample/card-styling-sample.component.scss]: This snippet should use the **igx-color** function.

File: src/app/layouts/card/card-styling-sample/card-styling-sample.component.scss [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    $background:#fd6bd93a,
+    $header-text-color:#f2f607,
+    $subtitle-text-color: #fa2509,
+    $content-text-color:#0a41f7
+);
+
+/*ENHANCED APPROACHES*/
+
+// $yellow-color: #e7eb0b;
+// $black-color: #f00b1e;
```
Comment (tachojelev) [inline src/app/layouts/card/card-styling-sample/card-styling-sample.component.scss]: This color is not black. maybe we should rename the variable.

File: src/app/layouts/card/card-styling-sample/card-styling-sample.component.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<div class="sample-column card-wrapper">
```
Comment (tachojelev) [inline src/app/layouts/card/card-styling-sample/card-styling-sample.component.html]: We need a **config instance** for this sample in the card's config generator in order to run it in stackblitz.

File: src/app/layouts/card/card-styling-sample/card-styling-sample.component.scss [hunk 4]
```diff
@@ -0,0 +1,63 @@
+@import '~igniteui-angular/lib/core/styles/themes/index';
+
+.card-wrapper {
+    max-width: 320px;
+    min-width: 220px;
+}
+
+$dark-card: igx-card-theme(
+    $background:#fd6bd93a,
```
Comment (tachojelev) [inline src/app/layouts/card/card-styling-sample/card-styling-sample.component.scss]: We can format the colors to be consistent with the rest of the topics, for example:
**$background: #FD6BD93A** (space + uppercase)

File: src/app/layouts/layouts-routes-data.ts [hunk 5]
```diff
@@ -30,6 +30,7 @@ export const layoutsRoutesData = {
     "card-sample-2": { displayName: "Horizontal Card", parentName: "Card" },
     "card-sample-3": { displayName: "Horizontal Card Alt", parentName: "Card" },
     "card-sample-4": { displayName: "Other Examples", parentName: "Card" },
+    "card-styling-sample": { displayName: "Styled card", parentName: "Card" },
```
Comment (rmkrmk) [inline src/app/layouts/layouts-routes-data.ts]: Consider using a capital C for "card". All other card's samples are named "Card".

File: src/app/layouts/card/card-styling-sample/card-styling-sample.component.scss [hunk 6]
```diff
@@ -0,0 +1,63 @@
+@import '~igniteui-angular/lib/core/styles/themes/index';
+
+.card-wrapper {
+    max-width: 320px;
+    min-width: 220px;
+}
+
+$dark-card: igx-card-theme(
```
Comment (rmkrmk) [inline src/app/layouts/card/card-styling-sample/card-styling-sample.component.scss]: Please include the following comment as in the other styling samples: /*BASIC THEMING*/

Comment (astaev) [reply src/app/layouts/card/card-styling-sample/card-styling-sample.component.scss]: Done

File: src/app/layouts/layouts-routes-data.ts (hunk 5 above)
Comment (astaev) [reply src/app/layouts/layouts-routes-data.ts]: Done

File: src/app/layouts/card/card-styling-sample/card-styling-sample.component.scss (hunk 4 above)
Comment (astaev) [reply src/app/layouts/card/card-styling-sample/card-styling-sample.component.scss]: Done

File: src/app/layouts/card/card-styling-sample/card-styling-sample.component.scss (hunk 2 above)
Comment (astaev) [reply src/app/layouts/card/card-styling-sample/card-styling-sample.component.scss]: Done

File: src/app/layouts/card/card-styling-sample/card-styling-sample.component.scss (hunk 1 above)
Comment (astaev) [reply src/app/layouts/card/card-styling-sample/card-styling-sample.component.scss]: Done

File: src/app/layouts/card/card-styling-sample/card-styling-sample.component.html (hunk 3 above)
Comment (astaev) [reply src/app/layouts/card/card-styling-sample/card-styling-sample.component.html]: Done

Comment (astaev) [reply src/app/layouts/card/card-styling-sample/card-styling-sample.component.html]: fixed

**Pull request merged**